### PR TITLE
Buy pnk kleros poc

### PR DESCRIPTION
**Pull request by satello**
- Create new wrappers for RNG and PinakionPOC
- Call KlerosPOC for buyPinakion and balance

### CODE REVIEW

File: contract_wrapper/KlerosWrapper.js [hunk 1]
```diff
@@ -30,22 +30,45 @@ class KlerosWrapper extends ContractWrapper {
    * @return  truffle-contract Object | err The contract object or error deploy
    */
   deploy = async (
+      rngAddress = '0xB7669d5774Cc4E33a056B2A24A7c7B7957556b37',
```
Comment (n1c01a5) [inline contract_wrapper/KlerosWrapper.js]: I think it is better in the `.env.prod` and `.env.dev` files

File: contract_wrapper/KlerosWrapper.js [hunk 2]
```diff
@@ -30,22 +30,45 @@ class KlerosWrapper extends ContractWrapper {
...
+   * @param address contract address
+   * @return Conract Instance | Error
+   */
+  load = async (
+    address
+  ) => {
+    try {
+      const contractInstance = await this._instantiateContractIfExistsAsync(kleros, address)
+      this.contractInstance = contractInstance
+      this.address = address
```
Comment (n1c01a5) [inline contract_wrapper/KlerosWrapper.js]: CS for information can you add an empty line before the `return`

File: contract_wrapper/KlerosWrapper.js [hunk 3]
```diff
@@ -83,42 +106,76 @@ class KlerosWrapper extends ContractWrapper {
...
   buyPinakion = async (
     amount,
+    contractAddress, // address of KlerosPOC
     account = this._Web3Wrapper.getAccount(0)
   ) => {
-    // TODO make tx to smart contract
+    const contractInstance = await this.load(contractAddress)
+    let txHashObj
+    try {
+      txHashObj = await this.contractInstance.buyPinakion(
```
Comment (n1c01a5) [inline contract_wrapper/KlerosWrapper.js]: just `const txHashObj`

File: contract_wrapper/KlerosWrapper.js [hunk 4]
```diff
@@ -83,42 +106,76 @@ class KlerosWrapper extends ContractWrapper {
...
     let userProfile = await this._StoreProvider.getUserProfile(account)
     if (_.isNull(userProfile)) userProfile = await this._StoreProvider.newUserProfile(account)
+    return {
+      pending: userProfile.balance - contractBalance,
+      balance: contractBalance
+    }
+  }
+
+  getData = async (
+    contractAddress = '0x994bf7c41fe94000b2e388ed0a754FaD36dc5C0e',
```
Comment (n1c01a5) [inline contract_wrapper/KlerosWrapper.js]: if possible to the `env` files

File: contract_wrapper/PinakionWrapper.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+   * @return Conract Instance | Error
+   */
+  load = async (
+    address
+  ) => {
+    try {
+      const contractInstance = await this._instantiateContractIfExistsAsync(PinakionPOC, address)
+      this.contractInstance = contractInstance
+      this.address = address
+      return contractInstance
```
Comment (n1c01a5) [inline contract_wrapper/PinakionWrapper.js]: idem CS

File: contract_wrapper/PinakionWrapper.js [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+  transferOwnership = async (
+    contractAddress,
+    klerosAddress,
+    account = this._Web3Wrapper.getAccount(0)
+  ) => {
+    let txHashObj
+    try {
+      let contractInstance = await this.load(contractAddress)
+      txHashObj = await contractInstance.transferOwnership(
```
Comment (n1c01a5) [inline contract_wrapper/PinakionWrapper.js]: just `const txHashObj`

File: contract_wrapper/PinakionWrapper.js [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+      throw new Error(e)
+    }
+    return txHashObj.tx
+  }
+
+  getData = async (
+    contractAddress,
+    account = this._Web3Wrapper.getAccount(0)
+  ) => {
+    let contractInstance = await this.load(contractAddress)
```
Comment (n1c01a5) [inline contract_wrapper/PinakionWrapper.js]: use `const`

File: contract_wrapper/KlerosWrapper.js (hunk 1 above)
Comment (satello) [reply contract_wrapper/KlerosWrapper.js]: I will just delete these

**Pull request merged**